### PR TITLE
Clarify adding collaborator permissions

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR clarifies that orgs can't add collaborators to repos, but can grant access with teams and roles instead.

### Related issues (optional)

Closes JIRA: https://docker.atlassian.net/browse/ENGDOCS-1888

### CODE REVIEW

**Review by wglambert — APPROVED** [review]
👌

File: content/docker-hub/repos/access.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ repository from that repository's **Settings** page.
 > **Note**
 >
 > A collaborator can't add other collaborators. Only the owner of
-> the repository has administrative access.
+> the repository has administrative access. Also, organizations can't add collaborators. Organization owners can control repository access with member roles and teams. See [Roles and permissions](../../security/for-admins/roles-and-permissions.md).
```
Comment (craig-osterhout) [inline content/docker-hub/repos/access.md:21]: `Also, organizations can't add collaborators.`
I got hung up on this. Maybe the missing part is "organization repositories".
Something like, organization repositories can't have collaborators. Or, you can't add collaborators to organization repositories.

Comment (stephaurelio) [reply content/docker-hub/repos/access.md:21]: Makes sense, will update!

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**